packages/terra-functional-testing/lib/commands/run-axe.js

```diff
@@ -0,0 +1,38 @@
+/* global browser, axe */
+const injectAxe = require('./inject-axe');
+
+/**
+ * Executes axe on the browser.
+ * @param {Object} overrides - The axe options.
+ * @param {Array} overrides.rules - The rule overrides
+ */
+const runAxe = (overrides = {}) => {
+  // Extract the axe options for the Terra service from the global browser object.
+  const [, options = {}] = browser.options.services.find(([service]) => (
+    typeof service === 'function' && service.name === 'TerraService'
+  ));
+
+  const { axe: axeOptions } = options;
+  const isAxeUnavailable = browser.execute(() => window.axe === undefined);
+
+  // Inject axe-core onto the page if it has not already been initialize.
+  if (isAxeUnavailable) {
+    injectAxe(axeOptions);
+  }
+
+  // Merge the global rules and overrides together.
+  const rules = {
+    ...axeOptions && axeOptions.rules.reduce((acc, rule) => ({ ...acc, [rule.id]: rule }), {}),
+    ...overrides.rules,
+  };
+
+  // eslint-disable-next-line prefer-arrow-callback, func-names
+  return browser.executeAsync(function (opts, done) {
+    // eslint-disable-next-line prefer-arrow-callback, func-names
+    axe.run(document, opts, function (error, result) {
+      done({ error, result });
+    });
+  }, { rules, restoreScroll: true, runOnly: ['wcag2a', 'wcag2aa', 'wcag21aa', 'section508'] });
```

Comment:
do we need to add `wcag21a` to our standards? i'm actually confused on how these keys translate to what standard we run against. if `wcag2aa` contains WCAG 2.0 and WCAG 2.1 Level AA, do we need to include `wcag21aa` then? or is that for explicit purposes

Reply:
The way these standards work is each tag has a set of rules that apply to that tag. So each tag here concats a new list of rules to be run. The rules don't need to be unique, so they can be in multiple standards but duplicates get filtered out. To add a new standard will add a new set of rules. 

Here's a little more info that helped me understand how this works:

https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#parameters

https://github.com/dequelabs/axe-core/blob/develop/doc/rule-descriptions.md

As far as adding a new standard I think that's worth a discussion. Adding a standard is non-passive because it adds a new set of rules, but we can be non-passive here if we want to.

Reply:
ahh gotcha. the docs are confusing - the website lists the `wcag2a` and `wcag2aa` tags as also containing the WCAG2.1 standards


<img width="763" alt="Screen Shot 2020-06-19 at 11 28 08 AM" src="https://user-images.githubusercontent.com/14063284/85156631-f4434100-b21f-11ea-9597-2d53708ff807.png">